demos/projects/ST/b-l475e-iot01a/port/sockets_wrapper_stm32l475.c

```diff
@@ -0,0 +1,615 @@
+/*
```

Comment:
I believe this breaks with the convention of the other TCP stacks that live in a more generic location (./ports or whatever I'm too lazy to lookup:().  I think that's right given the other TCP are pretty generic whereas this one isn't.

You might consider (consider only) some sort of documentation reference in say like the "./ports" of like "Oh by the way, we have a sample of ST networking too under this directory.

Reply:
https://github.com/Azure-Samples/iot-middleware-freertos-samples/pull/7#discussion_r647080608